src/queries/dataRequests/internal/monthly-delivery-report.sql

```diff
@@ -198,6 +198,39 @@ WITH
     JOIN "UserRoles" ur ON u.id = ur."userId"
     JOIN "Roles" r ON ur."roleId" = r.id
   ),
+  filtered_grants AS (
+    SELECT DISTINCT
+      gr.*
+    FROM "Grants" gr
+    LEFT JOIN "GrantReplacements" grr
+    ON gr.id = grr."replacedGrantId"
+    WHERE gr."deleted" != true
+	AND (grr."replacementDate" IS NULL
+        OR grr."replacementDate" > '2020-08-31')
+    -- Filter for startDate dates between two values if ssdi.startDate is defined
+    AND (NULLIF(current_setting('ssdi.startDate', true), '') IS NULL
+        OR gr."inactivationDate" IS NULL
+        OR gr."inactivationDate"::date >= (
+        SELECT MIN(value::timestamp)::date
+        FROM json_array_elements_text(COALESCE(NULLIF(current_setting('ssdi.startDate', true), ''),'[]')::json) AS value
+        ))
+    AND (NULLIF(current_setting('ssdi.startDate', true), '') IS NULL
+        OR gr."startDate"::date <= (
+        SELECT MAX(value::timestamp)::date
+        FROM json_array_elements_text(COALESCE(NULLIF(current_setting('ssdi.startDate', true), ''),'[]')::json) AS value
+        ))
+    AND (NULLIF(current_setting('ssdi.startDate', true), '') IS NULL
+        OR gr."endDate"::date >= (
```

Comment:
We're okay with the Grants that have null `endDate` being excluded, correct? If so, we should note it, or if not we'll have to add an `OR gr."endDate" IS NULL`

Reply:
all endDate's are on inactive grants that have not been updated since 04/2023. So I don't think it will be an issue.